FordLabs/PeopleMover
# My role warning

## Issue
Resolves #432

## What was done
- [x] Change the text of warning for traits when a trait already exists
- [x] Submit button for a trait is disabled when a trait already exists and warning appear while user are typing 
## How to test
1. Verify verbage of warning.
2. Try to create a new role with the same name as an existing role and see the warning show up and the button gets disabled

Feature URL: https://featurebranchui.apps.pd01i.edc1.cf.ford.com/
Feature Branch Deployment: https://jenkins-fordlabs.apps.pd01e.edc1.cf.ford.com/job/PeopleMover/job/Deploy%20PeopleMover%20FeatureBranch/
